website/docs/r/mysql_server.html.markdown

```diff
@@ -86,9 +75,23 @@ Possible values for `storage_mb` when using a SKU Name of `Standard` are:
 
 * `sku` supports the following:
 
-* `name` - (Optional) Specifies the SKU Name for this MySQL Server. Possible values are: `MYSQLB50`, `MYSQLB100`, `MYSQLS100`, `MYSQLS200`, `MYSQLS400` and `MYSQLS800`.
-* `capacity` - (Optional) Specifies the DTU's for this MySQL Server. Possible values are `50` and `100` DTU's when using a `Basic` SKU and `100`, `200`, `400` or `800` when using the `Standard` SKU.
-* `tier` - (Optional) Specifies the SKU Tier for this MySQL Server. Possible values are `Basic` and `Standard`.
+    * `name` - (Required) Specifies the SKU Name for this MySQL Server. The name of the SKU, follows the `tier` + `family` + `cores` pattern (e.g. B_Gen4_1, GP_Gen5_8). For more information see the [product documentation](https://docs.microsoft.com/en-us/rest/api/mysql/servers/create#sku).
```

Comment:
What's the impact for old MySQL resource provisioned by old schema? Any migration work if upgrade to future new provider?

Reply:
None, it is a no op for the end user. Please see the [Migration Results](https://gist.github.com/jeffreyCline/e3842f082c50ad15e3eaea2388154cd7).